docs/web3.eth.rst

```diff
@@ -200,6 +200,18 @@ The following methods are available on the ``web3.eth`` namespace.
         >>> web3.eth.get_storage_at('0x6C8f2A135f6ed072DE4503Bd7C4999a1a17F824B', 0)
         '0x00000000000000000000000000000000000000000000000000120a0b063499d4'
 
+.. py:method:: Eth.blob_base_fee
+
+    Fetches the expected base fee for blobs in the next block.
+    * Delegates to ``eth_blobBaseFee`` RPC Method
+
+    Returns the expected base fee in wei, represented as a hexadecimal.
```

Comment:
> represented as a hexadecimal.

This may be true of the JSON-RPC spec, but the library returns the formatted value as an integer. Perhaps we can leave this simpler, borrowing from the `max_priority_fee_per_gas` description. We also need a blank line between the first sentence and the bullet (*) in order to render the docs properly.

>    Fetches the expected base fee for blobs in the next block.
> 
>    * Delegates to ``eth_blobBaseFee`` RPC Method
>
>    Returns the expected blob base fee in Wei.

This renders well

<img width="712" alt="Screenshot 2024-11-14 at 10 54 25" src="https://github.com/user-attachments/assets/74d950ed-7c77-4150-9fc4-087e9dd43347">

Thoughts?

Reply:
sounds reasonable. I'll fix it